packages/react-native-windows-init/package.json

```diff
@@ -38,7 +38,7 @@
     "@types/chalk": "^2.2.0",
     "@types/jest": "^26.0.20",
     "@types/node": "^14.14.22",
-    "@types/prompts": "^2.4.0",
+    "@types/prompts": "2.0.10",
```

Comment:
This is intentionally a lower semver. Someone made the minor version lower for newer typings, so semver latest brings in an old version instead of latest.